### PR TITLE
fix: login button text margin

### PR DESCRIPTION
**Pull request by ayaankhan98**
Issue #346 
is it fine or any other modification?
button

### CODE REVIEW

File: lib/studies/rally/login.dart [hunk 1]
```diff
@@ -397,7 +397,7 @@ class _FilledButton extends StatelessWidget {
       style: TextButton.styleFrom(
         backgroundColor: RallyColors.buttonColor,
         primary: Colors.black,
-        padding: const EdgeInsets.symmetric(vertical: 10, horizontal: 24),
+        padding: const EdgeInsets.symmetric(vertical: 14, horizontal: 24),
```
Comment (perclasson) [inline lib/studies/rally/login.dart:400]: As mentioned in the ticket, the smaller Rally login button on desktop is probably due to [`VisualDensity`](https://api.flutter.dev/flutter/material/VisualDensity-class.html) being set to `adaptivePlatformDensity` and thus `compact` for Rally in desktop.

With your change the Button would become bigger on Android and iPhone phones. We will want to set the `visualDensity` to [`VisualDensity.standard`](https://api.flutter.dev/flutter/material/VisualDensity/standard-constant.html) instead.

To do this update the `ThemeData` for Rally: https://github.com/flutter/gallery/blob/2900cb93aeed58c953bda570774894ec7d30b08c/lib/studies/rally/app.dart#L58

**Comment by ayaankhan98** [comment]
@perclasson after setting the `visualDensity` to `standard` this is what we have

#### Linux
linux

#### Web (Chrome)
chrome

everything looks good except the `Text`, `Text` should be at the same base level as that of `lock icon`. To fix this should i set the margin again in `lib/studies/rally/login.dart`?

**Comment by perclasson** [comment]
> @perclasson after setting the `visualDensity` to `standard` this is what we have
> 
> 
> 
> #### Linux
> 
> linux
> 
> 
> 
> #### Web (Chrome)
> 
> chrome
> 
> 
> 
> everything looks good except the `Text`, `Text` should be at the same base level as that of `lock icon`. To fix this should i set the margin again in `lib/studies/rally/login.dart`?
> 
> 

This looks good. I think we do not need to modify the base level of the icon and text👍🏻

**Comment by ayaankhan98** [comment]
okay no problem :)

**Review by perclasson — APPROVED** [review]
LGTM, thanks for the fix and dealing with the back and forth!

**Pull request merged**